### ISSUE TITLE
IE11 Performance

### ISSUE THREAD

**Issue by fredfortier** [SitePen/dgrid#1122]
Our QA team reported some performance issues with dgrid 0.4 and large result sets (say 2000). The performance is really bad just scrolling through the results (it takes 3-4 seconds to respond).

After more investigation, it appears that the problem is limited to workstations with rudimentary GPUs or VMs with "integrated graphics". It is snappy on a gaming-type workstation with a high-performance GPU. It is also snappy on other browsers (Chrome/Firefox) regardless of the workstation.

It seems like some aspects of dGrid may not play nice with some of IE hardware acceleration "features". But I do not know where to look for. Have you experienced similar issues? Any pointer would be appreciated.

**Comment by dylans** [comment]
Are you using OnDemandGrid? I assume you're not actually trying to render all 2000 rows of data? What other mixins are you including?

**Comment by fredfortier** [comment]
Yes. I am using OnDemandGrid. My page size is 200. An implementation of dstore/Store controls the result set.

The behavior manifests itself with any kind of attempt at scrolling (even a notch down). It does not occur when fetching the next page. It seems to be related to the scroller, not fetching data.

Here is a snippet of my grid definition:
`var Grid = declare([OnDemandGrid, Keyboard, DijitRegistry, ColumnResizer, Selector, ColumnReorder], {
        showHeader: true,
        showFooter: true,
        minRowsPerPage: 50,
        maxRowsPerPage: 200,
        bufferRows: 200,
        cellNavigation: false,
        selectionMode: this.selectionMode,
});`
